revisions.js

```diff
@@ -145,4 +218,14 @@ const isMojganHere = classMates
 console.log(isMojganHere);
 
 // Exercise: Reduce the array of students int a a string of first nam and age seperated by commas. Ex: Nic is 32, Naime is 29...
-// Do this same task in *TWO* different ways
+// Do this same task in *TWO* different ways.
+
+const firstString = classMates.reduce((result, student) => {
+  return result + `${student.firstName} is ${student.age}, `;
+}, "");
+
+console.log(firstString);
+
+// Second solution:
+const stringMaker = (result, student) => result + `${student.firstName} is ${student.age}, `;
+console.log(classMates.reduce(stringMaker, ""));
```

Comment:
I wonder if we could do something to this array so that we could just `Array.prototype.join` it at the end :thinking: